plaso/cli/helpers/parsers.py

```diff
@@ -26,21 +26,20 @@ def AddArguments(cls, argument_group):
       argument_group (argparse._ArgumentGroup|argparse.ArgumentParser):
           argparse group.
     """
-    # TODO: rename option name to parser_filter_expression.
     argument_group.add_argument(
         '--parsers', dest='parsers', type=str, action='store',
-        default='', metavar='PARSER_LIST', help=(
-            'Define a list of parsers to use by the tool. This is a comma '
-            'separated list where each entry can be either a name of a parser '
-            'or a parser list. Each entry can be prepended with an '
-            'exclamation mark to negate the selection (exclude it). The list '
-            'match is an exact match while an individual parser matching is '
-            'a case insensitive substring match, with support for glob '
-            'patterns. Examples would be: "reg" that matches the substring '
-            '"reg" in all parser names or the glob pattern "sky[pd]" that '
-            'would match all parsers that have the string "skyp" or "skyd" '
-            'in its name. All matching is case insensitive. Use "--parsers '
-            'list" or "--info" to list the available parsers.'))
+        default='', metavar='PARSER_FILTER_EXPRESSION', help=(
```

Comment:
Mabye: ', or list available parsers' => ', or show the available parsers and plugins.'

Reply:
> Preset matching is exact, while parser and plugin matching is case insensitive

Don't you want to change the preset filtering to be case insensitive? or parser / plugin name filtering to be case sensitive instead?

"list" here does not refer to the Python type but list used in general, since this is help description. Hence I opt:
```
Define which parser or plugin to use or show the available parsers and plugins. The parser filter filter expression is a comma separated list of parser and plugin names, where each element is the name of a parser, plugin or preset. An element can be excluded by pre-pending it with an exclamation mark for example "!bash_history". ...
```

Reply:
Also, it turns out all matching in case insensitive.

Reply:
ack, more clear now, thx for fixing